src/lib/CollectStore.js

```diff
@@ -145,7 +145,7 @@ export default class CollectStore {
               arguments: '0 0 0 * * *',
               worker: 'konnector',
               worker_arguments: {
-                konnector: connection.konnector.attributes.slug,
+                konnector: connection.konnector.slug,
```

Comment:
So if it still works the first time it is ok for me 👍 

As a missing slug is an unexpected error and a blocker issue, shouldn't we throw an error instead of console.warn ? Also, `connection` is always defined in this part of the code.